andrepereiradasilva/joomla-cms
# Normalize "show_associations" and "flags" article parameters across com_content xmls

Pull Request for New issue.

### Summary of Changes

There are two parameters that where introduced in https://github.com/joomla/joomla-cms/pull/12042 that only exist in some com_content xmls.

This PR intends to put that parameters in all com_content xml that have the article params, so they can configured there like any other article param. 

### Testing Instructions

1. Apply patch
2. In the following panels confirm you have "Show Associations" and the "Use Image Flags" parameters:
 - "Articles" -> "Options" button, "Articles" tab
 - "Content" -> "Add New Article", "Options" tab
 - "Menus" -> "Add new menu item"
   - "Articles" -> "Archived Articles" type, "Options" tab
   - "Articles" -> "Category Blog" type, "Options" tab
   - "Articles" -> "Category List" type, "Options" tab
   - "Articles" -> "Featured Articles" type, "Options" tab
   - "Articles" -> "List All Categories" type, "Options" tab
   - "Articles" -> "Single Article" type, "Options" tab
   - "Articles" -> "Single Article" type, "Options" tab

### Documentation Changes Required

None
